include/video.h

```diff
@@ -188,18 +188,18 @@ enum class PixelFormat : uint8_t {
 	// FFmpeg Equivalent: AV_PIX_FMT_BGR24
 	BGR24_ByteArray = 24,
 
-	// 16.7M (32-bit) true colour; 8 bits per red/blue/green component;
-	// stored as packed uint32 data with highest 8 bits unused
+	// 16.7M (32-bit) true colour, 8 bits per red/blue/green component;
+	// stored as packed 32-bit data
 	//
-	// Stored as array of uint32_t in host native endianess.
-	// Each uint32_t is layed out as follows:
-	// (msb)8X 8R 8G 8B(lsb)
+	// Stored as array of uint8_t in BGRA memory order (endian agnostic)
 	// Example:
-	// uint32_t pixel = (unused << 24) | (red << 16) | (green << 8) | (blue << 0)
+	// uint8_t *pixels = image.image_data;
+	// pixels[0] = blue; pixels[1] = green; pixels[2] = red; pixels[3] = alpha;
 	//
-	// SDL Equivalent: SDL_PIXELFORMAT_XRGB8888
-	// FFmpeg Equivalent: AV_PIX_FMT_0RGB32
-	XRGB8888_Packed32 = 32
+	// SDL Equivalent: SDL_PIXELFORMAT_BGRA32 (alias for SDL_PIXELFORMAT_BGR24)
+	// FFmpeg Equivalent: AV_PIX_FMT_BGR0
+	// OpenGL Equivalent: GL_BGRA with data type GL_UNSIGNED_INT_8_8_8_8_REV
+	BGR32_ByteArray = 32,
```

Comment:
So is this `BGRX32` or `BGRA32`? We must specify the meaning of fourth component as well, as per above.

Reply:
As per above, I've concluded this is in fact `BGRX32_ByteArray` 😄